expressions/aggregation/AggregationHandleMin.hpp

```diff
@@ -134,6 +134,13 @@ class AggregationHandleMin : public AggregationConcreteHandle {
       const AggregationStateHashTableBase &hash_table,
       std::vector<std::vector<TypedValue>> *group_by_keys) const override;
 
+  AggregationState* aggregateOnDistinctifyHashTableForSingle(
```

Comment:
It is the implementation of `AggregationHandle::aggregateOnDistinctifyHashTableForSingle()`.
Perhaps just brief note here 
`See AggregationHandle::aggregateOnDistinctifyHashTableForSingle() for documentation`?